checks/shell_download_validate.go

```diff
@@ -480,22 +482,28 @@ func isUnpinnedPipInstall(cmd []string) bool {
 			continue
 		}
 
-		// TODO(laurent): https://github.com/ossf/scorecard/pull/611#discussion_r660203476.
-		// Support -r <> --require-hashes.
+		// https://github.com/ossf/scorecard/issues/1306#issuecomment-974539197.
+		if strings.EqualFold(cmd[i], "--require-hashes") {
+			hashHashes = true
+			continue
+		}
 
 		// Exclude *.whl as they're mostly used
 		// for tests. See https://github.com/ossf/scorecard/pull/611.
 		if strings.HasSuffix(cmd[i], ".whl") {
 			hasWhl = true
```

Comment:
We don't actually care whether `.whl` files are present or not, just that they don't trigger `hasUnresolvedDeps`.
```suggestion
```